setup.py

```diff
@@ -766,6 +770,12 @@ def print_box(msg):
         packages=packages,
         entry_points=entry_points,
         install_requires=install_requires,
+        package_dir={
+            'torch.testlib' : './test',
+            'torch.testlib.data' : './test/data',
+            'torch.testlib.expect' : './test/expect',
+            'torch.testlib.distributed.rpc' : './test/distributed/rpc'
+        },
```

Comment:
The use of `package_dir` for the entirety of the `test` directory looks very suspicious to me. I guess the reasoning is, you didn't want to move the `test` folder? Better say so...

Reply:
I'm sorry for changing my mind about this. I don't think it is a good idea to use the `package_dir` trick here. Just go ahead and move the files. Only the files that are imported will get moved, is that right? That will limit the blast radius of the move, since those files churn less than the real test files. What I am very afraid of is that people we see `import torch.testing._internal`, they will go to `torch/testing/_internal` and they will be like "Wut, where are the files?" I really don't want to have to explain this to every new person who comes and works on the codebase.